commands/operator-sdk/cmd/test/cluster.go

```diff
@@ -153,7 +154,11 @@ func testClusterFunc(cmd *cobra.Command, args []string) error {
 			if err != nil {
 				return fmt.Errorf("test failed and failed to get error logs")
 			}
-			defer readCloser.Close()
+			defer func() {
+				if err := readCloser.Close(); err != nil && !fileutil.IsClosedError(err) {
+					log.Printf("Failed to close pod log reader: (%v)", err)
```

Comment:
Should this log statement not be `log.Error`?

Reply:
Yup, I'll fix that